File: public/phase-2-20160620022203/C/xor-sum.c
// Name : XOR sum
#include <stdio.h>
int main() {
	// input:
	// n :  the number of elements in the array
	// a :  integers from the array
	int n, a[100], xor_sum=3, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `xor_sum`

	// output
	printf("%d", xor_sum);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/xor-sum.java
// Name : XOR sum
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n :  the number of elements in the array
		// a :  integers from the array
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `xor_sum`

		// output
		// Dummy Data
		int xor_sum=3;

		System.out.println(xor_sum);

	}
}